src/cloud-api-adaptor/install/overlays/docker/kustomization.yaml

```diff
@@ -18,6 +18,7 @@ configMapGenerator:
     literals:
       - CLOUD_PROVIDER="docker"
       - CLOUD_CONFIG_VERIFY="false" # It's better set as true to enable could config verify in production env
+      - DISABLECVM="true" # set as false to enable confidential VM
```

Comment:
Previously it was not set and the `docker` handler did not add any option, therefore it always used to run without the `-disable-cvm` argument. With this change it will, by default, add this argument. To keep the behaviour I'd suggest leaving this commented-out, what do you think?

Reply:
Yeah, I don't think the docker provider will do anything with the flag, but it makes sense to keep the default behaviour the same